components/raftstore/src/store/worker/split_check.rs

```diff
@@ -186,8 +185,8 @@ where
         }
     }
 
-    /// Checks a Region with split checkers to produce split keys and generates split admin command.
-    fn check_split(&mut self, region: &Region, auto_split: bool, policy: CheckPolicy) {
+    /// Checks a Region with split and bucket checkers to produce split keys and buckets keys and generates split admin command.
+    fn check_split_and_bucket(&mut self, region: &Region, auto_split: bool, policy: CheckPolicy) {
```

Comment:
should change `region_split_check_diff` when the region is enlarged, otherwise it would seldom trigger this check.

Reply:
better to calculate a reasonable default value for `region_split_check_diff`, no one want to change it.